src/components/ResultsLayout.js

```diff
@@ -455,18 +462,22 @@ function ResultsLayout({ data, triggerClickAnalytics, isPreview }) {
                             item,
                             get(resultSettings, 'fields.price'),
                         );
-
-                        const redirectToProduct = !isPreview || handle;
-
                         const { variants } = item;
 
+                        const redirectToProduct = !isPreview || handle;
+                        let url = '';
+                        if(redirectToProduct) {
+                            if(handle.includes('http')) {
```

Comment:
@lohi212 Use specifically: `http://` or `https://`, otherwise this would match something like: `/http-product-1234`